framework/actions/input.go

```diff
@@ -35,12 +35,15 @@ type InputContext struct {
 	rulesStore rules.Store
 	// Settings define adjustable rule settings parsed from framework-specific implementation
 	Settings settings.Settings
+	// action concurrency
+	MaxConcurrentWeight int64
 }
 
 // NewContext returns an InputContext for the given OSCAL Components.
 func NewContext(components []components.Component) (*InputContext, error) {
 	inputCtx := &InputContext{
-		requestedProviders: make(map[plugin.ID]string),
+		requestedProviders:  make(map[plugin.ID]string),
+		MaxConcurrentWeight: 3,
```

Comment:
Could be make this limit configurable in runtime?

Reply:
Thanks for the feedback @marcusburghardt. To clarify, you mean as an option in the CLI or the config?

Reply:
Added this option and the timeout value as config options